api/main.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import FastAPI, File, UploadFile
+
+app = FastAPI()
+
+
+@app.get("/")
+def read_root():
+    return {"This is": "a test"}
+
+
+@app.post("/upload/")
+async def upload(file: UploadFile):
+    contents = await file.read()  # --> file is a standard python file object
```

Comment:
@Lioncat2002 Could you make sure that the file being accepted here is an image? If it isn't an image, the api should respond accordingly